src/layouts/login-form.html

```diff
@@ -53,6 +53,7 @@
             id='passwordInput'
             label="[[localize('ui.login-form.password')]]"
             type='password'
+            autofocus='autofocus'
```

Comment:
`autofocus` should be enough (without `='autofocus'`)

Reply:
I figured it wouldn't hurt (was taught that XHTML didn't like boolean attributes in short form, and didn't check the doctype to be honest).

Reply:
For which browser is this fix btw?
There is already code for autofocus and it's fine Chrome and FIrefox

Reply:
I'm using Chrome and FF here, and I'm not sure which one caused the issue, but I had the page open multiple times without focussing the pw field.